Packs/ApiModules/Scripts/HTTPFeedApiModule/HTTPFeedApiModule.py

```diff
@@ -341,7 +347,7 @@ def fetch_indicators_command(client, itype, **kwargs):
                         "rawJSON": attributes,
                     }
 
-                    if len(client.custom_fields_mapping.keys()) > 0:
+                    if len(client.custom_fields_mapping.keys()) > 0 or 'tags' in attributes.keys():
```

Comment:
lets make one list that contain the keys and `'tags'` and then iterate it
`'tags'` should be const